### PR TITLE
Add AWS 9.1.1 release notes

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/8664

### CODE REVIEW

File: aws.yaml [hunk 1]
```diff
@@ -55,8 +55,8 @@
       version: 0.7.0
     - name: cluster-operator
       provider: aws
-      version: 0.22.1-dev
-  date: 2020-02-03T12:00:00Z
+      version: 0.22.1
+  date: 2020-02-07T12:00:00Z
```
Comment (rossf7) [inline aws.yaml]:
```suggestion
  date: 2020-02-04T12:00:00Z
```

Is the date wrong?

File: release-notes/aws/v9.1.1.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## :zap: Giant Swarm Release 9.1.1 for AWS is now active for you! :zap:
+
+### cluster-operator [v0.22.1](https://github.com/giantswarm/cluster-operator/releases/tag/v0.22.1)
+
+- No change from the previous version.
+
+### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
+
+- Allow disabling TCP DNS check.
```
Comment (cokiengchiara) [inline release-notes/aws/v9.1.1.md]:
```suggestion
- Allowed disabling TCP DNS check.
```

Comment (cokiengchiara) [reply release-notes/aws/v9.1.1.md]: Any way the change makes customer's life better? (Figuring out if should write sth for paragraph at top)

File: release-notes/aws/v9.1.1.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## :zap: Giant Swarm Release 9.1.1 for AWS is now active for you! :zap:
+
```
Comment (cokiengchiara) [inline release-notes/aws/v9.1.1.md]:
```suggestion
This release upgrades the NGINX IC app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.
```

Comment (cokiengchiara) [reply release-notes/aws/v9.1.1.md]: WDYT about this? 
Also, what can the customer do here - any CTA we want to add? Like, should they upgrade, talk to SE... or just fyi?

**Comment by cokiengchiara** [comment]
@sslavic thanks! Do you know if there be changes from other teams in this Release? 

[that's what our [process](https://github.com/giantswarm/giantswarm/blob/master/processes/release-organization-communication.md#steps) says to ask, but it's kind of circular... how can i coordinate for you, if i am asking you who i should coordinate with. 😅

File: release-notes/aws/v9.1.1.md (hunk 2 above)
Comment (sslavic) [reply release-notes/aws/v9.1.1.md]: It made our life better, it's a fix for https://github.com/giantswarm/giantswarm/issues/7898 - our monitoring tool made use of it and was erroring/alarming on one customer environment where this is not permitted.

File: release-notes/aws/v9.1.1.md (hunk 3 above)
Comment (sslavic) [reply release-notes/aws/v9.1.1.md]: Applied suggestion as is, likely IC abbreviation should be replaced with Ingress Controller.
Even more likely we need to add a note related to https://github.com/giantswarm/giantswarm/issues/8851#issuecomment-583723774 - can you please come up with a suggestion? Or would you like me to propose initial content for the note? Same needs to be communicated for https://github.com/giantswarm/releases/pull/89

File: release-notes/aws/v9.1.1.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.
```
Comment (oponder) [inline release-notes/aws/v9.1.1.md]: One of the sentences reads a bit weird for me, here's a suggestion.

```suggestion
This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also enabled the metrics Service for prometheus-operator. This allows us to scrape for metrics and improves our monitoring the app.
```

**Comment by sslavic** [comment]
@cokiengchiara I've updated this AWS legacy WIP release notes to cover ingress-nginx 0.29.0 upgrade and default SSL ciphers change. Please take another look and let me know what you think.

File: release-notes/aws/v9.1.1.md (hunk 4 above)
Comment (sslavic) [reply release-notes/aws/v9.1.1.md]: I've done some wordsmithing, PTAL

File: aws.yaml [hunk 5]
```diff
@@ -103,8 +103,8 @@
       provider: aws
       version: 2.0.0
 - version: 9.1.1
-  active: false
-  date: 2020-02-03T12:00:00Z
+  active: true
```
Comment (sslavic) [inline aws.yaml]: Will move activating 9.1.1 to a separate PR.

**Comment by sslavic** [comment]
@cokiengchiara any objections to merging 9.1.1 release notes as they are? Additional PRs can continue to be made. I'll create separate PR for activating the release - until release is active (but even after activating) release notes are considered as WIP.

**Comment by cokiengchiara** [comment]
> @cokiengchiara any objections to merging 9.1.1 release notes as they are? Additional PRs can continue to be made. I'll create separate PR for activating the release - until release is active (but even after activating) release notes are considered as WIP.

LGTM. If I could double approve, I will.

**Pull request merged**